charts/data-space-connector/values.yaml

```diff
@@ -627,6 +626,113 @@ keycloak:
+  ## configuration to be used by every api-deployment if nothing specific is provided.
+  defaultConfig:
+    # --  ngsi-ld broker connection information
+    ngsiLd:
+      # -- address of the broker
+      url: http://data-service-scorpio:9090
+    contextUrl: https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld
+
+  # enable the api proxy
+  apiProxy:
+    enabled: true
+    image:
+      tag: distroless-v1.27-latest
+    service:
+      # -- name to be used for the proxy service.
+      nameOverride: tm-forum-api
+
+  # -- list of apis to be deployed
+  # -- every api needs to define a name and image. Basepath is required when routes or ingress will be enabled. Beside that, they can overwrite the default-config by using the same keys.
+  # -- be aware: when you change the image repositrory or the tag for an api, you have to provide both values for the changes to take effect
+  apis:
+    - name: party-catalog
+      image: tmforum-party-catalog
+      basePath: /tmf-api/party/v4
+
+    - name: customer-bill-management
+      image: tmforum-customer-bill-management
+      basePath: /tmf-api/customerBillManagement/v4
+
+    - name: customer-management
+      image: tmforum-customer-management
+      basePath: /tmf-api/customerManagement/v4
+
+    - name: product-catalog
+      image: tmforum-product-catalog
+      basePath: /tmf-api/productCatalogManagement/v4
+
+    - name: product-inventory
+      image: tmforum-product-inventory
+      basePath: /tmf-api/productInventory/v4
+
+    - name: product-ordering-management
+      image: tmforum-product-ordering-management
+      basePath: /tmf-api/productOrderingManagement/v4
+
+    - name: resource-catalog
+      image: tmforum-resource-catalog
+      basePath: /tmf-api/resourceCatalog/v4
+
+    - name: resource-function-activation
+      image: tmforum-resource-function-activation
+      basePath: /tmf-api/resourceFunctionActivation/v4
+
+    - name: resource-inventory
+      image: tmforum-resource-inventory
+      basePath: /tmf-api/resourceInventoryManagement/v4
+
+    - name: service-catalog
+      image: tmforum-service-catalog
+      basePath: /tmf-api/serviceCatalogManagement/v4
+
+  # redis caching
+  redis:
+    enabled: false
+
+  registration:
+    enabled: true
+    # -- name to be used for the registration jobs
+    name: tmf-api-registration
+    # -- configuration to register the dataplane at the credentials-config-service
+    ccs:
+      # -- endpoint of the ccs to regsiter at
+      endpoint: http://credentials-config-service:8080
+      # -- service id of the data-service to be used
+      id: tmf-api
+      # -- default scope to be created for the data plane
+      defaultOidcScope:
+        # -- name of the scope
+        name: default
+        # -- name of the default credential to be configured
+        credentialType: VerifiableCredential
+        # -- needs to be updated for the concrete dataspace
+        trustedParticipantsLists: http://tir.trust-anchor.org
```

Comment:
I know that the documentation already says that value needs to be replaced, but maybe if we change the domain to something even more unrealistic, it might be useful ( I can image receiving issue requests saying "tir.trust-anchor.org is down" and submitting some log files)